resources/assets/uad_lists.json

```
@@ -6437,7 +6437,7 @@
   {
     "id": "com.samsung.android.applock",
     "list": "Oem",
-    "description": "Samsung App Lock\nLets you lock your app (Settins > Advanced fuctions > App lock)\nYou should lock your apps storing private data (provides data at rest encryption when your phone is locked)\n",
+    "description": "Samsung App Lock\nLet's you lock your app (Settins > Advanced fuctions > App lock)\nYou should lock your apps storing private data (provides data at rest encryption when your phone is locked)\n",
```

```suggestion
    "description": "Samsung App Lock\nLets you lock your app (Settins > Advanced fuctions > App lock)\nYou should lock your apps storing private data (provides data at rest encryption when your phone is locked)\n",
```
Incorrect. "Lets" is correct: https://www.grammarly.com/blog/lets-vs-lets/#:~:text=The%20difference%20between%20let's%20and%20lets&text=Let's%20is%20a%20contraction%20of%20%E2%80%9Clet%20us.%E2%80%9D%20You%20use,to%20allow%20or%20give%20permission.